Security/AddDigitalSignature/AddDigitalSignature.vb

```diff
@@ -0,0 +1,67 @@
+Imports Datalogics.PDFL
+
+'
+'
+' This sample shows how to add a Digital Signature
```

Comment:
Can we add a period at the end of the sample description to be consistent with other samples?

Reply:
If you want to change the samples, we'd need to make a separate PR to develop. This is the PR to merge the already-approved changes to the main branch.